doc/manual.rst

```diff
@@ -2708,6 +2708,23 @@ a parameter typed as `untyped` (for unresolved expressions) or the type class
   assert toSeq2(items(@[1,2])) == @[1, 2] # but items(@[1,2]) is
 
 
+Overload disambiguation
+=======================
+
+There is a weaker form of overload disambiguation performed when an overloaded
```

Comment:
Weaker than what? What is this referring to?

Reply:
good point. weaker than "overload resolution" which kicks in for routine calls. Will update.